### ISSUE TITLE
Fix confusing "Passthrough" flashing method description

### ISSUE THREAD

**Issue by jurgelenas** [ExpressLRS/ExpressLRS-Configurator#698]
When you have a TX module connected directly through the USB-C cable to your PC this "Passthrough" flashing method description looks really confusing:

![image](https://github.com/user-attachments/assets/afaa75af-5530-4349-9916-5becd17140ad)

**Comment by L1cardo** [comment]
Looks like this is Radiomaster related issuse.
![image](https://github.com/user-attachments/assets/e33758a1-e366-4017-b3c5-ebdd5c068488)

**Comment by pkendall64** [comment]
> Looks like this is Radiomaster related issue.

No, they are two different ways of flashing the backpack! UART means that the module has switches to connect the backpack to the USB connector. Passthrough means that the serial data is passed through the main MCU to the backpack for flashing.

**Comment by L1cardo** [comment]
What I am trying to say is that for other TXs there is not "Passthrough" option, only "UART" and "WiFi".
I only see "Passthrough" for Radiomaster TXs

**Comment by pkendall64** [comment]
![Screenshot 2024-11-18 at 1 53 39 PM](https://github.com/user-attachments/assets/390840c0-9535-4653-b8aa-4fe18d92729d)

BETAFPV (newer modules) has it, as well as EMAX. The Passthrough option is only available for external modules that have the correct passthrough circuitry.

**Comment by L1cardo** [comment]
So the problem is, when you connect a TX via USB port, there should be a "UART" option.
So I don't think the "Passthrough " description is confusing.
What makes it confusing is not the  description, is the missing "UART" option

**Comment by pkendall64** [comment]
But you can't do UART flashing of the backpack on those modules!
So you are just saying that you think the label should be UART (or better USB) as the option for UART and Passthrough, even though the underlying mechanism is different.

**Comment by L1cardo** [comment]
Emmm, I know the underlying mechanism.
I am the one who is really confused right now😂

The title of this issue says "Passthrough description is confusing when you plug the TX via USB port", if you connect via USB, there is nothing to do with Passthrough. 

So you're saying change the description to something like "connect via USB port is not passthrough, connect your TX to your radio to use Passthrough method", right?

**Comment by pkendall64** [comment]
The current description for passthrough is confusing. It needs to say something like "Flash the backpack via the USB port on the TX module".
It's called passthrough because it puts the ESP32 into passthrough mode to flash the backpack. This is different to "ETX Passthrough", where the data is passed through the handset/radio to the internal module and then to the backpack.

**Comment by L1cardo** [comment]
I got it know, so there are two Passthrough for backpack, one is USB Passthrough, the other is Edgetx Passthrough.
